File: RootMJ/Models/UserProfile.cs
namespace RootMJ.Models
{
    public class UserProfile
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string ImageLocation { get; set; }
        public int NeighborhoodId  { get; set; }
        public string FirebaseUserId { get; set; }
        public bool IsAdmin { get; set; }
        public Neighborhood Neighborhood { get; set; }

    }
}

File: RootMJ/client/src/components/UserProfile.js
import React from "react";
import { Card, CardBody } from "reactstrap";

export const UserProfile = ({ userProfile }) => {
    return (
        <Card className="m-4">
            <CardBody>
                <p>
                    <img src={userProfile.imageLocation} />
                </p>
                <strong>{userProfile.name}</strong>
                <p>{userProfile.email}</p>
                <p>{userProfile.neighborhood.name}</p>

            </CardBody>
        </Card>
    );
}